script/app-mlperf-inference-mlcommons-python/_cm.yaml

```diff
@@ -1018,6 +1018,7 @@ variations:
     group: models
     env:
       CM_MODEL: mixtral-8x7b
+      CM_ML_MODEL_STARTING_WEIGHTS_FILENAME: "https://github.com/anandhu-eng/cm4mlops/blob/5f156eb891b86a4ea7f1864075ddf1f1ade66598/script/get-ml-model-mixtral/_cm.json#L66"
```

Comment:
We should add this from the mlcommons repo

Reply:
Updated: https://github.com/mlcommons/cm4mlops/pull/353/commits/7163f9b765de01301b624c536f1d4c8a24c16f98